dist/angular-svg-radial.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * angular-svg-radial 0.1.0
+ * (c) 2015 Vokal LLC
+ * https://github.com/vokal/angular-svg-radial
+ */
+angular.module("svgRadial", []).directive("radialProgress", function() {
+    "use strict";
+    return {
+        restrict: "E",
+        transclude: true,
+        replace: true,
+        template: '<div class="radial-progress">' + '<div class="base"></div>' + '<div class="radial">' + '<div class="piece"></div>' + '<div class="piece"></div>' + "</div>" + '<div class="transclude" ng-transclude></div>' + "</div>",
```

Comment:
Should that move to `/source` be now? I was debating doing that since this isn't currently in use, but I wasn't sure how it would influence a version number shift (if at all since there has yet to be a release)?

The third file may feel superfluous at the moment due to its similarities with the source code. Eventually there will be additional documentation that will be removed in this step.